docs/items/weapon-configs.md

```diff
@@ -142,4 +142,450 @@ What values a Weapon Type can have:
+* Config: Combo
+* Ammo: Default
+  * 100%
+* REQ Teir: 5
+* Game State: 4
+* Notes:-
+  
+## [18] FUStalker
+* Weapon Type: Stalker Rifle + Striker Sidekick
+* Trait Set: 18
+  * Weapon Damage: 2.70
+* Config: Combo
+* Ammo: Default
+  * 100%
+* REQ Teir: 7
+* Game State: 6
+* Notes:-
+
+## [19] Bayonet Ravager
+* Weapon Type: Ravager + Deulist Energy Sword
+* Trait Set: 19
+  * Weapon Damage: 1.0
+* Config: Combo
+* Ammo: Default
+  * 100%
+* REQ Teir: 5
+* Game State: 4
+* Notes:-
+
+## [20] Provoker
+* Weapon Type: Ravager + Elite Bloodblade
+* Trait Set: 20
+  * Weapon Damage: 1.61
+* Config: Combo
+* Ammo: Default
+  * 100%
+* REQ Teir: 7
+* Game State: 6
+* Notes:-
+
+## [21] Charged Disruptor
+* Weapon Type: Disruptor + Rushdown Hammer
+* Trait Set: 21
+  * Weapon damage: 1.87
+* Config: Combo
+* Ammo: Default
+  * 12+32
+* REQ Teir: 5
+* Game State: 4
+* Notes:-
+
+## [22] Charged Shock Rifle
+* Weapon Type: Shock Rifle + Diminisher Of Hope
+* Trait Set: 22
+  * Weapon Damage: 1.50
+* Config: Combo
+* Ammo: Cut (61)
+  * 12+17
+* REQ Teir: 7
+* Game State: 6
+* Notes:-
+
+## [23] H1 Mangler
+* Weapon Type: Mangler + Stalker Rifle Ultra
+* Trait Set: 23
+  * Weapon Damage: 0.9
+* Config: Combo
+* Ammo: Default
+  * 8+16
+* REQ Teir: 5
+* Game State: 4
+* Notes:-
+
+## [24] Binary Beam
+* Weapon Type: Sentinel Beam + Impact Commando
+* Trait Set: 24
+  * Weapon Damage: 0.46
+* Config: Combo
+* Ammo: Default
+  * 40+120
+* REQ Teir: 5
+* Game State: 4
+* Notes:-
+
+## [25] FUBeam
+* Weapon Type: Sentinel Beam + Striker Sidekick
+* Trait Set: 25
+  * Weapon Damage: 0.52
+* Config: Combo
+* Ammo: Default
+  * 14+42
+* REQ Teir: 7
+* Game State: 6
+* Notes:-
+
+## [26] Guidedshot
+* Weapon Type: Cindershot + M41 Tracker
+* Trait Set: 26
+  * Weapon Damage: 1.10
+* Config: Combo
+* Ammo: Default
+  * 6+3
+* REQ Teir: 7
+* Game State: 6
+* Notes:-
+
+## [27] Fast Skewer
+* Weapon Type: Skewer + Diminisher Of Hope
+* Trait Set: 27
+  * Weapon Damage: 1.05
+  * Movement Speed: 1.20
+  * Jump Height: 1.30
+  * Reload Speed
+    * Empty Reload Scalar: 3.00
+    * Tactical Reload Scalar: 3.00
+* Config: Combo
+* Ammo: Default
+  * 1+3
+* REQ Teir: 6
+* Game State: 5
+* Notes:-
+
+## [28] M247-Jumper
+* Weapon Type: M247 HMG + Rushdown Hammer
+* Trait Set: 28
+  * Weapon Damage: 1.0
+* Config: Combo
+* Ammo: Default
+  * 100%
+* REQ Teir: 4
+* Game State 3
+* Notes: Has a speed and jump boost from Rushdown Hammer
+
+## [29] M247-Striker
+* Weapon Type: M247 HMG + Striker Sidekick
+* Trait Set: 29
+  * Weapon Damage: 0.3
+  * Movement Speed With Turret: 1.20
+* Config: Combo
+* Ammo: Default
+  * 100%
+* REQ Teir: 5
+* Game State: 4
+* Notes:-
+  
+## [30] Jorge's Chaingun
+* Weapon Type: M247 HMG + MA40 Longshot
+* Trait Set: 30
+  * Weapon Damage: 1.2
+  * Movement Speed With Turret: 1.47
+  * Damage Resistance
+    * Direct Damage Scalar: 2.10
+    * Grenade Damage Scalar: 1.0
+    * Explosive Damage Scalar: 1.0
+* Config: Scripted
```

Comment:
Add more granular info about the scripted gun's features. A template for this is provided starting at line 9.